src/mapbox/extended_geolocate_control.js

```diff
@@ -21,7 +30,10 @@ class ExtendedGeolocateControl extends GeolocateControl {
 
   trigger() {
     window.navigator.permissions && window.navigator.permissions.query({ name: 'geolocation' }).then(p => {
-      if (p.state === 'prompt') {
+      if(p.state === geolocationPermissions.GRANTED) {
+        Telemetry.add(Telemetry.LOCALISE_START)
```

Comment:
This event will not be triggered when prompt is used (on the first visit for example)